### ISSUE TITLE
OpenAPI Generator Errors

### ISSUE THREAD

**Issue by grokify** [grokify/go-ringcentral-client#23]
This issue is to keep track of the compile time errors encountered with [`openapi-generator`](https://github.com/OpenAPITools/openapi-generator/issues/new), a fork of Swagger Codegen.

```
../../client/api_meetings.go:39:25: undefined: optional
../../client/api_presence.go:566:29: undefined: optional
../../client/api_ring_out.go:178:18: undefined: optional

../../client/model_pool_response_resource.go:34:21: undefined: os

../../client/api_call_handling_settings.go:646:5: undefined: localVarFile
../../client/api_call_handling_settings.go:647:28: undefined: localVarFile
../../client/api_call_handling_settings.go:649:22: undefined: localVarFile
../../client/api_call_handling_settings.go:650:3: undefined: localVarFile
../../client/api_glip.go:82:5: undefined: localVarFile
../../client/api_glip.go:83:28: undefined: localVarFile
../../client/api_glip.go:85:22: undefined: localVarFile
../../client/api_glip.go:86:3: undefined: localVarFile
../../client/api_messages.go:649:5: undefined: localVarFile
../../client/api_messages.go:650:28: undefined: localVarFile

../../client/model_throwable.go:12:6: invalid recursive type Throwable
```

**Comment by grokify** [comment]
`api_meetings.go`

```yaml
      post:
        tags:
          - "Meetings"
        summary: "Create Meetings"
        operationId: "createMeeting"
        consumes:
          - "application/json"
        produces:
          - "application/json"
        parameters:
          -
            in: "body"
            name: "body"
            required: false
            schema:
              $ref: "#/definitions/MeetingRequestResource"
```

```go
import (
	"context"
	"fmt"
	"io/ioutil"
	"net/http"
	"net/url"
	"strings"
)

// Linger please
var (
	_ context.Context
)

type MeetingsApiService service

/*
MeetingsApiService Create Meetings
 * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 * @param extensionId
 * @param accountId
 * @param optional nil or *CreateMeetingOpts - Optional Parameters:
 * @param "MeetingRequestResource" (optional.Interface of MeetingRequestResource) -
@return MeetingResponseResource
*/

type CreateMeetingOpts struct {
	MeetingRequestResource optional.Interface
}

func (a *MeetingsApiService) CreateMeeting(ctx context.Context, extensionId string, accountId string, localVarOptionals *CreateMeetingOpts) (MeetingResponseResource, *http.Response, error) {
	var (
```

**Comment by grokify** [comment]
Fixed in f434cfbe99ff558cfe04a9d8ebd7cfb90a38c12f
